src/components/utilities/color/colors.config.yml

```diff
@@ -0,0 +1,43 @@
+label: colors
+
+context:
+  colors:
+    color:
+      - violet
+      - coastal
+      - indigo
+      - marine
+      - blue
+      - sky
+      - cyan
+      - teal
+      - mint
+      - green
+      - lime
+      - citron
+      - yellow
+      - gold
+      - orange
+      - peach
+      - terra
+      - red
+      - magenta
+      - purple
+      - gray-cool
+      - gray
+      - gray-warm
```

Comment:
@thisisdano mentioned he needs to change the color names, so this would need to change.